### DISCUSSION TITLE
Hi! I'm a designer.

### DISCUSSION THREAD

**Discussion by peteseyeview** [FundamentalFrequency/.github#37] [UI/UX]
I saw another thread about sending SVGs for UI/UX. Where would I find a base set to work from?

**Comment by stonepreston** [comment]
The UI is done completely in code, so there is no base set really. For icons, use [FontAwesome](https://fontawesome.com). If you would like to provide mock ups or ideas of UI improvements, feel free to post them in this section of the discussion board.
